UnityProject/Assets/Scripts/Items/Implants/Organs/Bones.cs

```diff
@@ -23,14 +24,8 @@ public override void ImplantPeriodicUpdate()
 		{
 			if ((RelatedPart.HealthMaster.CirculatorySystem.StartingBlood * GenerationOvershoot) > RelatedPart.HealthMaster.CirculatorySystem.BloodPool.Total)  //Assuming this is blood cap max)
 			{
-				float toConsume = RelatedPart.PassiveConsumptionNutriment * RelatedPart.HealingNutrimentMultiplier;
-				if (toConsume > RelatedPart.HealthMaster.CirculatorySystem.BloodPool[RelatedPart.Nutriment])
-				{
-					toConsume = RelatedPart.HealthMaster.CirculatorySystem.BloodPool[RelatedPart.Nutriment];
-				}
-
-				RelatedPart.HealthMaster.CirculatorySystem.BloodPool.Remove(RelatedPart.Nutriment, toConsume);
-				RelatedPart.HealthMaster.CirculatorySystem.BloodPool.Add(GeneratesThis, BloodGeneratedByOneNutriment * toConsume);
+				RelatedPart.HungerConsumption = hungerUsedNormal * RelatedPart.HealingNutrimentMultiplier;
```

Comment:
this just generates even if you have no food?

Reply:
hm?

Reply:
Ah right, oversight, Ill correct that